plugins/snapshots/blockfile/blockfile.go

```diff
@@ -49,6 +50,9 @@ type SnapshotterConfig struct {
 	// mountOptions are the base options added to the mount (defaults to ["loop"])
 	mountOptions []string
 
+	// copySparse determine whether sparse files when copy (defaults to true，equivalent to "cp --sparse=always xxx")
```

Comment:
Where is this defaulting to true? We also would not want this to be the default since it is not necessary on all file systems.